core/vm/runtime/runtime_types.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_VM_RUNTIME_RUNTIME_TYPES_HPP
+#define CPP_FILECOIN_CORE_VM_RUNTIME_RUNTIME_TYPES_HPP
+
+#include "common/buffer.hpp"
+#include "primitives/address/address.hpp"
+#include "primitives/big_int.hpp"
+#include "vm/exit_code/exit_code.hpp"
+
+namespace fc::vm::runtime {
+
+  using actor::MethodNumber;
+  using actor::MethodParams;
+  using common::Buffer;
+  using exit_code::ExitCode;
+  using primitives::BigInt;
+  using primitives::address::Address;
+
```

Comment:
here and below add descriptions to types

Reply:
done

Reply:
done